README.md

```diff
@@ -169,19 +190,21 @@ logging long tests.
 user=> (run-tests (find-tests "test") {:test-warn-time 500})
 ```
 
-### Plugin
+### Leiningen Plugin
 
 To use the Lein-Eftest plugin, just run:
 
 ```sh
 lein eftest
 ```
 
-You can customize the reporter and set the concurrency by adding an
-`:eftest` key to your project map:
+You can customize the reporter, set the concurrency mode, set the
```

Comment:
Change this to:

    You can customize the reporter and configure the concurrency settings
    by adding an `:eftest` key to your project map: